packages/core/src/application.ts

```diff
@@ -147,6 +153,10 @@ export class Application extends Context {
       const options = OPENAPI_SPEC_MAPPING[request.url];
       return this._serveOpenApiSpec(request, response, options);
     }
+    if (request.method === 'GET' && request.url &&
+        request.url === '/swagger-ui') {
+      return this._renderOpenApiSpec(request, response);
```

Comment:
nitpick: perhaps `_redirectToSwaggerUI` would be a better name?

Reply:
Yeah I think that's a more descriptive name. 👍